src/components/SkillCardScrollList/SkillCardScrollList.js

```diff
@@ -18,12 +18,36 @@ class SkillCardScrollList extends Component {
     this.state = {
       cards: [],
       skills: this.props.skills,
+      scrollCards: 3,
     };
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     this.loadSkillCards();
-  }
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+
+    $('.scrolling-wrapper').css({ width: '104%' });
```

Comment:
Why is this needed?

Reply:
This was done to make the arrow icon look better on full width, but I think I'll remove this from here and take care of this along with the other UI issues in https://github.com/fossasia/susi_skill_cms/issues/946. Thanks :-)